### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by astrojuanlu**
Didn't touch the intro and Kedro notebooks yet, I'll send that in another PR.

### CODE REVIEW

**Comment by astrojuanlu** [comment]
@deepyaman Should we `rm -r quarto`? The docs there are out of sync now

**Comment by astrojuanlu** [comment]
Also @deepyaman I started a new Codespace on this branch and somehow `$CODESPACE_VSCODE_FOLDER` wasn't defined, any clue?

File: demo/delay-prediction/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,8 @@
+duckdb==0.10.2
```
Comment (deepyaman) [inline demo/delay-prediction/requirements.txt:1]: I don't think this is necessary? Unless we need just that version. Actually already have a bit duplicative requirements, in `ibis-framework` and the Kedro-Datasets extras.

File: demo/delay-prediction/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+ibis-framework[duckdb,polars,bigquery,examples]==9.0.0
```
Comment (deepyaman) [inline demo/delay-prediction/requirements.txt:2]: Can we drop BigQuery? I saw you're using some Polars in your examples, but I think dropping BigQuery could lose the other Google deps in `codespace_requirements.txt` eventually.

File: demo/delay-prediction/conf/base/parameters_data_science.yml [hunk 3]
```diff
@@ -1,12 +0,0 @@
...
-  random_state: 3
-  features:
-    - engines
-    - passenger_capacity
-    - crew
-    - d_check_complete
-    - moon_clearance_complete
-    - iata_approved
-    - company_rating
-    - review_scores_rating
```
Comment (astrojuanlu) [inline demo/delay-prediction/conf/base/parameters_data_science.yml:12]: These parameters weren't being used anywhere, adding new ones

Comment (deepyaman) [reply demo/delay-prediction/conf/base/parameters_data_science.yml:12]: Sounds good

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,6 @@
+-r demo/delay-prediction/requirements.txt
+
 altair==5.3.0
-duckdb==0.10.2
-ibis-framework[duckdb,polars,bigquery,examples]==9.0.0
 jupyterlab==4.1.8
 plotnine==0.13.5
 plotly>=5.22.0
```
Comment (deepyaman) [inline requirements.txt:6]: I'm actually going to drop the ecosystem notebook, so hopefully can simplify this, too.

File: demo/delay-prediction/src/delay_prediction/pipelines/training/__init__.py
Comment (deepyaman) [inline demo/delay-prediction/src/delay_prediction/pipelines/training/__init__.py:1]: Nit: Should we do `model_training` and `model_evaluation`, to be consistent with `data_processing`?

**Review by deepyaman — APPROVED** [review]
Saw you're still editing; in any case, some quick notes, but all looks good, and feel free to merge whenever you're ready.

File: demo/delay-prediction/requirements.txt (hunk 1 above)
Comment (astrojuanlu) [reply demo/delay-prediction/requirements.txt:1]: Fair enough, I was moving this around!

File: demo/delay-prediction/requirements.txt (hunk 2 above)
Comment (astrojuanlu) [reply demo/delay-prediction/requirements.txt:2]: We don't need it for this indeed 🏃🏼

File: demo/delay-prediction/src/delay_prediction/pipelines/training/__init__.py
Comment (astrojuanlu) [reply demo/delay-prediction/src/delay_prediction/pipelines/training/__init__.py:1]: Good idea, changing

**Pull request merged**